appstate/jaeger_state.go

```diff
@@ -0,0 +1,12 @@
+package appstate
+
+import "github.com/kiali/kiali/config"
+
+// JaegerEnabled is a memo-flag to tell if Jaeger was configured or succesfully discovered
+var JaegerEnabled = false
```

Comment:
@aljesusg Currently I see we have 3 different flags that serve similar purposes with however some slight differences: there's `JaegerEnabled`, `JaegerAvailable` and `TracingConfig.Enable`. I wonder if we could simplify that.

Reply:
yes, @jotak I know.... I don't know how added JaegerEnabled...let me introduce the flags:

## Enabled

User set this in the configuration to enable or disable jaeger in the UI (like displaylink of grafana)

## jaegerAvailable

This is a internal configuration for us, in the discover step we check if kiali discovered the service and endpoint or the user set this information, if all is ok this variable will be true. So if there is a service with route all is conmfigured.

## jaegerEnabled 

This variable is false if kiali can't reach http://<jaeger>/api/services in business.GetServices().

So **jaegerEnabled** check if the application is available and up and **jaegerAvailable** if there is a tracing service with a route defined.

Here is the problem. jaegerEnabled is ok because the user can have the correct setting in his environment but for some reason jaeger is not available, for example elasticsearch is wrong. Should we warn the user about this? there is so use cases here.